test/cql-pytest/test_describe.py

```diff
@@ -2567,3 +2568,90 @@ def aux(row: DescRow) -> DescRow:
             assert restore_stmts == res
         finally:
             cql.execute(f"DROP KEYSPACE IF EXISTS {ks}")
+
+### ===========================================================================
+
+# `varchar` is an alias for `text`. Scylla defaults to `text` and we test it here.
+# It's not very important, but it's better to have a test that detects changes
+# like that and verifies we handle that case as expected.
```

Comment:
I had to read the code below to understand what you are actually testing here. Basically you are saying that when you DESC FUNCTION a function that refers to a "varchar" type, DESC FUNCTION prints out the alias "text" instead of "varchar". This test verifies that this this bizarre (but harmless, since the two type names are aliases) transformation remains.

My only question, is why... Why do we *want* this silly renaming to never change? What if in the future we print "text" for "text" - why would that be bad or wrong in any way?

Finally, why is this test specific to UDF and DESC FUNCTION? What about DESC TABLE which can have a varchar field? What about UDAs? collections of varchars? Why don't we test varchar there? 

I see we have a complicated get_random_type() which tries very hard to be random and forgot to cover all the types ;-)